### ISSUE TITLE
SAC_AWMP agent

### ISSUE THREAD

**Issue by Acedorkz** [hzm2016/SAC-AWMP#1]
Hi, sorry to interrupt,
I ran your SAC_AWMP on Half-Cheetah env,
(line 139: phi_loss.backward(retain_graph=True)  # 2. Back propagation)
but got 'Runtime Error: one of the variables needed for gradient computation has beed modified by an inplace operation'
Can u help me with this?
Thanks a lot

**Comment by www2171668** [comment]
I meet the same problem as @Acedorkz .
Do you deal with this problem?

**Comment by www2171668** [comment]
This problem also happen in SAC

**Comment by www2171668** [comment]
知道怎么弄了，在算法文件中调整backward的顺序；或者在目标网络输出值后面加detach()控制梯度，并在policy_loss.backard()之前再输出一次qf1_pi和qf2_pi。跑了几个程序，SAC是复合跑分的，但SAC_AWMP远远没跑出那个效果
